howto/get-secrets/README.md

```diff
@@ -35,7 +35,7 @@ import (
 )
 
```

Comment:
It would be good to add in the docs [here](https://github.com/dapr/docs/tree/master/howto/get-secrets#calling-the-secrets-api) that following code snippets shows fetching secret from default namespace stored in kubernetes secret store